src/shared/components/cancerSummary/CancerSummaryContent.tsx

```diff
@@ -466,10 +466,12 @@ export class CancerSummaryContent extends React.Component<ICancerSummaryContentP
                         href={this.pngAnchor} download="cBioPortalCancerSummary.png">
                         PNG <i className="fa fa-cloud-download" aria-hidden="true"></i>
                     </a>
+                    /*
```

Comment:
have you confirmed this works? I'm pretty sure this is invalid JSX and you need to wrap in enclosing curly brackets

Reply:
@adamabeshouse you are right, good tip about the curly braces. The errors you can make in three lines of code :)